src/components/MoviePage/MoviePage.css

```diff
@@ -17,6 +17,19 @@ button {
   margin: 0;
 }
 
+.genre {
+  padding: 8px;
+  border: 1px solid white;
+  border-radius: 18px;
+}
+
+.genre-container {
+  display: flex;
+  justify-content: space-around;
```

Comment:
Loving these CSS properties! Space-around and space-between are making the comp look great!